docs/logagent/cli-parameters.md

```diff
@@ -91,9 +91,9 @@ The default output is line-delimited JSON for parsed log lines, as long as no fo
 |LOGS_TMP_DIR | Directory to store failed bulk requests for later retransmission.|
 |LOG_INTERVAL | Time to batch logs before a bulk request is done. Default is 10000 ms (10 seconds)|
 |LOGS_BULK_SIZE | Maximum size of a bulk request. Default is 1000.|
-|LOGS_RECEIVER_URL | URL for the Logsene receiver. For a local Elasticsearch server or for Sematext Enterprise version of Logsene. Defaults to Sematext Logsene SaaS receiver https://logsene-receiver.sematext.com/_bulk. Example for Elasticsearch: ```LOGSENE_URL=https://localhost:9200/_bulk```|
+|LOGS_RECEIVER_URL | URL for the Logsene receiver. For a local Elasticsearch/Opensearch server or for Sematext Enterprise version of Logsene. Defaults to Sematext Logsene SaaS receiver https://logsene-receiver.sematext.com/_bulk. Example for Elasticsearch: ```LOGSENE_URL=https://localhost:9200/_bulk```|
```

Comment:
or for Sematext Enterprise version of Logsene. Defaults to Sematext Logsene SaaS receiver
==> 
Defaults to Sematext Logs receiver
(and replace "Logsene" in "Logsene receiver")